core/src/main/scala/org/apache/spark/ui/jobs/AllStagesPage.scala

```diff
@@ -172,6 +185,20 @@ private[ui] class AllStagesPage(parent: StagesTab) extends WebUIPage("") {
           {completedStagesTable.toNodeSeq}
         </div>
     }
+    if (shouldShowSkippedStages) {
+      content ++=
+        <span id="skipped" class="collapse-aggregated-allSkippedStages collapse-table"
```

Comment:
I see things like `collapse-aggregated-allCompletedStages` are mentioned in `webui.js`, so probably this one should be added there.

This is kinda past the point where this block should be refactored (it's repeated 5 times now), but probably ok for a 2.3 change.

Reply:
thanks, I added the UT. I think we can create a new JIRA/followup PR to resolve the code redundancy issue you rightly pointed out, what do you think?

Reply:
Sounds good to clean things up in a follow-up.